core/include/detray/geometry/volume_connector.hpp

```diff
@@ -20,6 +20,7 @@ namespace detray {
 /// @param volume_grid [in] the indexed volume grid
 ///
 template <typename detector_t,
+          typename scalar_t /* detray::scalar */,
```

Comment:
Hm, maybe it would be a good idea to define the scalar type in the detector, like ```typename scalar_type = scalar``` and access it here through the detector type. The rational is, that the scalar type should always match the scalar type used in the detector. What do you think?